src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -565,6 +565,32 @@ private sealed class ReceiveMessageFromOperation : ReadOperation
                 Callback!(BytesTransferred, SocketAddress!, SocketAddressLen, ReceivedFlags, IPPacketInformation, ErrorCode);
         }
 
+        private sealed unsafe class BufferPtrReceiveMessageFromOperation : ReadOperation
+        {
+            public byte* BufferPtr;
+            public int Length;
+            public SocketFlags Flags;
+            public int BytesTransferred;
+            public SocketFlags ReceivedFlags;
+            public IList<ArraySegment<byte>>? Buffers;
```

Comment:
Since this field is always null, we should rather remove it, and pass a constant `null` to `TryCompleteReceiveMessageFrom`.